src/main/java/eu/dissco/orchestration/backend/properties/ApplicationProperties.java

```diff
@@ -0,0 +1,18 @@
+package eu.dissco.orchestration.backend.properties;
+
+
+import jakarta.validation.constraints.NotBlank;
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@Validated
+@Component
+@ConfigurationProperties("application")
+public class ApplicationProperties {
+
+  @NotBlank
+  private String baseUrl;
```

Comment:
Where do you set this value?

Reply:
In the properties file and when deploying in the env variables file